docs/application/dotnet/guides/app-management/data-control.md

```diff
@@ -32,57 +32,65 @@ The main features of the Tizen.Applications.DataControl namespace include:
 
 ## Prerequisites
 
-The data control use cases involve 2 applications. Each application plays a different role: one as the consumer, the other as the provider.
+The data control use cases involve two applications. Each application plays a different role: one as a consumer, the other as a provider.
+
+To enable your application to use the data control functionality, you have to include [Tizen.Applications.DataControl](https://samsung.github.io/TizenFX/latest/api/Tizen.Applications.DataControl.html) namespace.
+```
+using Tizen.Applications.DataControl;
+```
+
+#### Consumer
+Consumer application needs to request specific permission by adding the following privileges to the `tizen-manifest.xml` file:
+```
+<privileges>
+    <privilege>http://tizen.org/privilege/datasharing</privilege>
+    <privilege>http://tizen.org/privilege/appmanager.launch</privilege>
+</privileges>
+```
+
+#### Provider
+
+For your provider, you need to include a few things in `tizen-manifest.xml`. If you are using Visual Studio, double-click `tizen-manifest.xml`, and in the manifest editor, go to **Advanced > Data Control**, and click **Add** to add the provider details. Add the `Read` and `Write` access rights to both `SQL` and `Map` types, as needed.
+
+You can set the data access to be trusted, allowing other applications signed with the same certificate to access the data. You can also define privileges to restrict access to applications having the specific privileges.
+
+If you are not using Visual Studio, this is sample `tizen-manifest.xml`, that you can modify for your needs accordingly.
+```
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns="http://tizen.org/ns/packages" api-version="8" package="org.tizen.example.ProviderSample" version="1.0.0">
+  <profile name="common" />
+  <service-application appid="org.tizen.example.ProviderSample"
+                       exec="ProviderSample.dll"
+                       multiple="false" 
+                       nodisplay="true" 
+                       taskmanage="false" 
+                       splash-screen-display="true" 
+                       type="dotnet">
+    <label>ProviderSample</label>
+    <icon>ProviderSample.png</icon>
+    <datacontrol providerid="http://org.tizen.example.ProviderSample/datacontrol/provider/Test" access="ReadWrite" type="Sql" />
+    <datacontrol providerid="http://org.tizen.example.ProviderSample/datacontrol/provider/Test" access="ReadWrite" type="Map" />
+    <background-category value="background-network" />
+  </service-application>
+  <privileges>
+    <privilege>http://tizen.org/privilege/datasharing</privilege>
+  </privileges>
+  <provides-appdefined-privileges />
+</manifest>
+
+```
+
+[!NOTE]
+In your consumer application, you must pass your `providerid` to consumer constructor methor. `providerid` is defined in the provider's `tizen-manifest.xml`. Add `.dll` to the end of `providerid` when you pass it to consumer constructor:
+```
```

Comment:
@MathewP1, minor changes suggested, plus you need to add the angle bracket sign as shown for Note style. Fix typo in metho**d**.

> [!NOTE] 
> In your consumer application, you must pass your `providerid` to the consumer constructor method. The `providerid` is defined in the provider's `tizen-manifest.xml`. Add `.dll` to the end of `providerid` when you pass it to the consumer constructor: